code/debris/debris.h

```diff
@@ -45,7 +45,7 @@ typedef struct debris {
 	int		model_instance_num;		// What model instance this uses - needed for arcs
 	int		submodel_num;			// What submodel this uses
 	int		next_fireball;			// When to start a fireball
-	int		is_hull;				// indicates a large hull chunk of debris
+	int		is_hull;				// indicates whether this is a collideable, destructable peice of debris from the model, or just a generic debris fragment
```

Comment:
It's spelled "piece"

Reply:
I don't think I will ever learn how to spell it properly.